--- error.cpp.old	2013-07-23 17:10:07.000000000 +0200
+++ error.cpp	2015-10-26 07:38:09.000000000 +0100
@@ -16,6 +16,7 @@
 #include "error.h"
 #include "universe.h"
 #include "output.h"
+#include "lammpsexception.h"
 
 using namespace LAMMPS_NS;
 
@@ -40,6 +41,8 @@
                                     "ERROR: %s (%s:%d)\n",str,file,line);
   }
 
+  throw LammpsException(file, str, line);
+
   if (output) delete output;
   if (universe->nworlds > 1) {
     if (screen && screen != stdout) fclose(screen);
@@ -61,6 +64,7 @@
   if (universe->uscreen)
     fprintf(universe->uscreen,"ERROR on proc %d: %s (%s:%d)\n",
             universe->me,str,file,line);
+  throw LammpsException(file, str, line);
   MPI_Abort(universe->uworld,1);
 }
 
@@ -95,6 +99,8 @@
     if (logfile) fprintf(logfile,"ERROR: %s (%s:%d)\n",str,file,line);
   }
 
+  throw LammpsException(file, str, line);
+
   if (output) delete output;
   if (screen && screen != stdout) fclose(screen);
   if (logfile) fclose(logfile);
@@ -121,6 +127,7 @@
     if (universe->uscreen)
       fprintf(universe->uscreen,"ERROR on proc %d: %s (%s:%d)\n",
               universe->me,str,file,line);
+  throw LammpsException(file, str, line);
   MPI_Abort(world,1);
 }
 
