--- fix_ave_time.h.org	2015-03-26 19:02:08.000000000 +0100
+++ fix_ave_time.h	2015-09-29 20:55:19.000000000 +0200
@@ -36,7 +36,7 @@
   double compute_scalar();
   double compute_vector(int);
   double compute_array(int,int);
-
+  FILE *fp;
  private:
   int me,nvalues;
   int nrepeat,nfreq,irepeat;
@@ -44,7 +44,7 @@
   int *which,*argindex,*value2index,*offcol;
   int *varlen;               // 1 if value is from variable-length compute
   char **ids;
-  FILE *fp;
+    
   int nrows;
   int any_variable_length;
   int all_variable_length;
